src/Microsoft.DocAsCode.Build.SchemaDriven/Models/DocumentSchema.cs

```diff
@@ -121,5 +107,95 @@ public static DocumentSchema Load(string schemaPath)
             }
         }
+
+        private static T LoadSchema<T>(JSchema schema, Dictionary<JSchema, BaseSchema> cache) where T : BaseSchema, new()
+        {
+            if (cache.TryGetValue(schema, out var bs))
+            {
+                return (T)bs;
+            }
+
+            bs = new T
+            {
+                Title = schema.Title,
+                Description = schema.Description,
+                Type = schema.Type,
+                Default = schema.Default,
+                ContentType = GetValueFromJSchemaExtensionData<ContentType>(schema, "contentType"),
+                Tags = GetValueFromJSchemaExtensionData<List<string>>(schema, "tags"),
+                MergeType = GetValueFromJSchemaExtensionData<MergeType>(schema, "mergeType"),
+                Reference = GetValueFromJSchemaExtensionData<ReferenceType>(schema, "reference"),
+                XrefProperties = GetValueFromJSchemaExtensionData<List<string>>(schema, "xrefProperties"),
+            };
+
+            cache[schema] = bs;
+
+            CheckForNotSupportedKeyword(schema.OneOf, nameof(schema.OneOf));
+            CheckForNotSupportedKeyword(schema.AllOf, nameof(schema.AllOf));
+            CheckForNotSupportedKeyword(schema.AnyOf, nameof(schema.AnyOf));
+            CheckForNotSupportedKeyword(schema.AdditionalItems, nameof(schema.AdditionalItems));
+            CheckForNotSupportedKeyword(schema.AdditionalProperties, nameof(schema.AdditionalProperties));
+            CheckForNotSupportedKeyword(schema.PatternProperties, nameof(schema.PatternProperties));
+
+            if (schema.Properties != null)
+            {
+                bs.Properties = new Dictionary<string, BaseSchema>();
+                foreach (var pair in schema.Properties)
+                {
+                    bs.Properties[pair.Key] = LoadSchema<BaseSchema>(pair.Value, cache);
+                }
+            }
+
+            if (schema.Items != null && schema.Items.Count > 0)
+            {
+                if (schema.Items.Count > 1)
+                {
+                    throw new SchemaFeatureNotSupportedException("Multiple item definition is not supported in current schema driven document processor");
+                }
+
+                bs.Items = LoadSchema<BaseSchema>(schema.Items[0], cache);
+            }
+
+            return (T)bs;
+        }
+
+        private static T GetValueFromJSchemaExtensionData<T>(JSchema schema, string key)
+        {
+            if (schema.ExtensionData != null
+                && schema.ExtensionData.TryGetValue(key, out var value))
+            {
+                return value.ToObject<T>();
+            }
+
+            return default(T);
+        }
+
+        private static void CheckForNotSupportedKeyword(object keyword, string name)
+        {
+            if (keyword == null)
+            {
+                return;
+            }
+
+            if (keyword is IList<JSchema> list)
+            {
+                if (list.Count > 0)
```

Comment:
>> [](start = 31, length = 1)

should be == ?

Reply:
`>0` means it contains value, however the feature is not supported.